tests/test_routes_githubWebHook_pull_request.py

```diff
@@ -116,7 +130,11 @@ def get_repo(self, repo_id):
         response = json.loads(rv.data.decode('utf-8'))
 
         self.assertEqual('All fine, thanks', response['info'])
-        logging.info.assert_called_with("execute_synchronize {'action': 'synchronize'}")
+        PullRequest_mock.create_issue_comment.assert_called_with('''This branch of this pull request was updated, times are reseted.
```

Comment:
Whatever you decide to change the other comment to this needs to be updated to match it.

Reply:
thx